### PR TITLE
Fix super->supertype deprecation warning

### PR DESCRIPTION
**Pull request by timholy**
I'm going to tag a new release, since the current one fails on 0.5 and makes packages like ProfileView fail their tests.

### CODE REVIEW

**Comment by lobingera** [comment]
how can deprecations warnings break testing?

**Comment by timholy** [comment]
They don't. I just wanted to fix test warnings before tagging.

**Comment by timholy** [comment]
The last tag of Gtk was in mid-June.

File: src/GLib/glist.jl [hunk 1]
```diff
@@ -16,9 +16,9 @@ immutable _GList{T} <: _LList{T}
     prev::Ptr{_GList{T}}
 end
 eltype{T}(::Type{_LList{T}}) = T
-eltype{L<:_LList}(::Type{L}) = eltype(super(L))
+eltype{L<:_LList}(::Type{L}) = eltype(supertype(L))
```
Comment (tkelman) [inline src/GLib/glist.jl]: is this in Compat 0.8.0 ?

File: .travis.yml [hunk 2]
```diff
@@ -3,10 +3,16 @@ sudo: false
 matrix:
   include:
     - os: linux
-      julia: 0.4
+      julia:
+        - 0.4
+        - 0.5
+        - nightly
```
Comment (tkelman) [inline .travis.yml]: I don't think it likes things being nested. Probably better to just set TESTCMD based on the output of uname

**Comment by timholy** [comment]
There's some Homebrew problem. Will this be exacerbated if I tag a new version, or is it already an issue now?

**Comment by tkelman** [comment]
ref https://github.com/JuliaLang/Homebrew.jl/issues/121 so it's been a problem for a while, but may be work-around-able with a `Homebrew.postinstall` call? cc @staticfloat so this doesn't fall off your radar completely

**Comment by timholy** [comment]
OK, so I won't let this hold up tagging.

**Pull request merged**